src/components/Modal.tsx

```diff
@@ -9,8 +9,6 @@ import Heading from './Heading';
 const StyledPaper = styled.div`
   background-color: ${({ theme }) => theme.palette.background};
   margin: 1rem;
-  min-width: 40rem;
-  max-width: 40rem;
```

Comment:
fikser nok scrollbaren, men hvis du gjør dette, må du nok gå igjennom alle modalene vi har og sette `maxWidth`-propen på alle sammen. Denne ser ikke bra ut nå f.eks:

![Skjermbilde 2020-06-12 kl  11 12 03](https://user-images.githubusercontent.com/6595982/84486333-92259180-ac9d-11ea-8da6-8cbc39dfaf63.png)

Reply:
ser ut som den blir bra hvis du setter `maxWidth="sm"` på `AuthorityModal`

Reply:
![Skjermbilde 2020-06-12 kl  11 12 03](https://user-images.githubusercontent.com/6595982/84494994-b6d43600-acaa-11ea-8375-157c0191afbf.png)

Denne bør kanskje også ha en minWidth satt? ser litt rart ut at den er så liten

Reply:
sikker på at du var på rett branch når du testa?
![bilde](https://user-images.githubusercontent.com/6313468/84498686-6b715600-acb1-11ea-9640-c02d022def36.png)
![bilde](https://user-images.githubusercontent.com/6313468/84498708-74fabe00-acb1-11ea-9022-082384c7fd2c.png)

"Legg til forfatter"-modalen ble ganske lang, men jeg syns det at modalen ikke endrer bredde når man får treff veier opp for det

Reply:
var på min branch, men kommenterte ut `min-width` og `max-width` i `Modal.tsx`. Mulig jeg hadde noe styling som påvirket det. Men grunnen til at den blir så stor til deg, er vel fordi du har satt `fullWidth`-propen?

Reply:
Det er jo egentlig pirk, og når vi får en designer, så vil det nok bli ytret en del om dette kan jeg tenke meg, så må sikkert uansett endres. Men fint om du sjekker at alle modalene vi har ser ok ut hvertfall da?

Reply:
<img width="1623" alt="Skjermbilde 2020-06-12 kl  13 44 29" src="https://user-images.githubusercontent.com/6595982/84499441-e0915b00-acb2-11ea-89df-5bcf8f9c22c5.png">

Tok ned din branch nå. Sånn ser det ut til meg

Reply:
har gått gjennom maxWidth på alle sånn at det ser bra nok ut etter min subjektive mening :)
Grunnen til at din koble til forfatteID blir rar er nok fordi du har en lang affiliation, og at hver rad med authority er fixed til `1fr 2fr 2fr`, og hvis vi endrer det vil det bli forferdelig i listen over mange authorities igjen 😅